tests/monotouch-test/VideoToolbox/VTPixelRotationSessionTests.cs

```diff
@@ -0,0 +1,77 @@
+//
+// Unit tests for VTPixelRotationSession
+//
+// Authors:
+//	Israel Soto <issoto@microsoft.com>
+//	
+//
+// Copyright 2022 Microsoft Corporation.
+//
+
+#if !__WATCHOS__
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Threading;
+
+using Foundation;
+using VideoToolbox;
+using CoreMedia;
+using CoreVideo;
+using AVFoundation;
+using CoreFoundation;
+using ObjCRuntime;
+using NUnit.Framework;
+using Xamarin.Utils;
+
+namespace MonoTouchFixtures.VideoToolbox {
+
+	[TestFixture]
+	[Preserve (AllMembers = true)]
+	public class VTPixelRotationSessionTests
+	{
+
+		[Test]
+		public void CreateTest ()
+		{
+			TestRuntime.AssertXcodeVersion (14, 0);
```

Comment:
Since all the tests need it to be in Xcode 14.0, you don't need to add it per test, add the following:
```csharp
[OneTimeSetUp]
public void Init () => TestRuntime.AssertXcodeVersion (14, 0);
```
That way, it will get called once for the entire test fixture